### ISSUE TITLE
Make releases?

### ISSUE THREAD

**Issue by clkao** [jupyterhub/wrapspawner#17]
So people can install this at with specific version number.

**Comment by minrk** [comment]
Yes, we should be publishing releases

**Comment by mbmilligan** [comment]
Agreed, we're overdue to push a release here. Since we don't have any CI testing in place yet, we have to rely on user reports to know if everything works. By now we do have a number of users though, so it definitely is in working condition at least for the current Jupyterhub release. How do we feel about the following plan:

1. Cut a release now, only claiming functionality against Jupyterhub 0.8.
2. Implement testing against additional environments.
3. Cut a new release that advertises confirmed functionality for a wider set of environments (possibly including Jupyterhub 0.9 by that time).
